java/developer-guide/presentation-slide/convert-slide/_index.md

```diff
@@ -3,144 +3,255 @@ title: Convert Slide
+
+``` java 
+Presentation pres = new Presentation("Presentation.pptx");
+try {
+    // Converts the first slide in the presentation to a Bitmap with the specified size
     BufferedImage bmp = pres.getSlides().get_Item(0).getThumbnail(new Dimension(1820, 1040));
-    {
-        // Save the image in JPEG format
-        ImageIO.write(bmp, "PNG", new File("Slide_0.jpg"));
-    }
+	
+	// Saves the image in the JPEG format
+	ImageIO.write(bmp, "PNG", new File("Slide_0.jpg"));
 } catch (Exception e) {
 } finally {
     if (pres != null) pres.dispose();
 }
 ```
 
-## **Convert Slide with Notes and Comments to Image**
+This Java code demonstrates how to convert the first slide to the framed image with the [RenderToGraphics](https://apireference.aspose.com/slides/java/com.aspose.slides/ISlide#renderToGraphics-com.aspose.slides.IRenderingOptions-java.awt.Graphics2D-java.awt.Dimension-) method:
 
-There are two interfaces [ITiffOptions](https://apireference.aspose.com/slides/java/com.aspose.slides/ITiffOptions) and [IRenderingOptions](https://apireference.aspose.com/slides/java/com.aspose.slides/IRenderingOptions), used to control the rendering of presentation slides to images.
-Both of these interfaces include the [INotesCommentsLayoutingOptions](https://apireference.aspose.com/slides/java/com.aspose.slides/INotesCommentsLayoutingOptions) interface, which can be used to include notes and comments of a slide in an exporting image.
-Using this interface, you can also control the position in which notes and comments will be displayed in the image.
-The following example demonstrates the usage of the [INotesCommentsLayoutingOptions](https://apireference.aspose.com/slides/java/com.aspose.slides/INotesCommentsLayoutingOptions) interface using the [IRenderingOptions](https://apireference.aspose.com/slides/java/com.aspose.slides/IRenderingOptions) interface.
-An example of ITiffOptions interface usage will be provided below. 
+``` java
+Presentation pres = new Presentation("Presentation.pptx");
+try {
+	java.awt.Dimension slideSize = new java.awt.Dimension(1820, 1040);
+
+	// Creates a BufferedImage with the specified size (slide size + fields)
+	BufferedImage slideImage = new BufferedImage(slideSize.width + 50, slideSize.height + 50, BufferedImage.TYPE_INT_ARGB);
+	java.awt.Graphics2D graphics = slideImage.createGraphics();
+	try {
+		// Fills and translates Graphics to create a frame around the slide
+		graphics.setColor(java.awt.Color.RED);
+		graphics.fillRect(-1, -1, slideImage.getWidth() + 1, slideImage.getHeight() + 1);
+		graphics.translate(12, 12);
+
+		// Renders the first slide to Graphics
+		pres.getSlides().get_Item(0).renderToGraphics(new RenderingOptions(), (Graphics2D) graphics, slideSize);
+	} finally {
+		if (graphics != null) graphics.dispose();
+	}
+
+	// Saves the image in the JPEG format
+	ImageIO.write(slideImage, "png", new File("FramedSlide_0.jpg"));
```

Comment:
Inconsistency in the output image format. Please choose one from JPEG or PNG.

Reply:
Done